Aptoide/aptoide-client-v8
# Feature/asv 2052 rakam 3 events

**What does this PR do?**

This PR implements 3 new events to rakam: click on install, download and install

**Database changed?**

   Yes

**Where should the reviewer start?**

- [ ] InstallAnalytics.java

**How should this be manually tested?**

Make sure the events are correctly sent to rakam (you can check the logs for a rakamEventLogger). You should see events for the three cases. Check the ticket and make sure they match the specification;

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-2052](https://aptoide.atlassian.net/browse/ASV-2052)

**Code Review Checklist**
